chia/wallet/util/debug_spend_bundle.py

```diff
@@ -41,7 +40,7 @@ def dump_coin(coin: Coin) -> str:
     return disassemble(coin_as_program(coin))
 
 
-def debug_spend_bundle(spend_bundle: SpendBundle, agg_sig_additional_data=bytes([3] * 32)) -> None:
+def debug_spend_bundle(spend_bundle: 'SpendBundle', agg_sig_additional_data=bytes([3] * 32)) -> None:
```

Comment:
I'm not sure this works if `SpendBundle` isn't also imported (but I haven't tried it). My skepticism arises from the "what does `mypy` do if there are two things named `SpendBundle` in the project and neither is imported?"

Also, `black` probably wants it to be double-quotes.

Reply:
I added a commit to remove this.  `mypy` does not like it, you are correct.  It can't be there because otherwise its a circular import.  The alternative is to move the whole method and its imports to the spend_bundle.py file which seems like a bit of a clutter.  This is a debugging tool literally named for what you're supposed to be passing it,so I don't think it's that big of a deal.